packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -590,6 +590,14 @@ export const AddIcon = (
   </svg>
 );
 
+export const BettingAddIcon = (
+  <svg viewBox="0 0 12 12" fill="none">
+    <path d="M6 3V9" stroke-width="2" stroke-miterlimit="10" stroke-linecap="square"/>
```

Comment:
these should be `strokeWidth` `strokeMiterlimit` and `strokeLinecap` respectively, same with the next line.